dot/network/notifications.go

```diff
@@ -238,7 +243,14 @@ func (s *Service) sendData(peer peer.ID, hs Handshake, info *notificationsProtoc
 		return
 	}
 
-	if support, err := s.host.supportsProtocol(peer, info.protocolID); err != nil || !support {
+	support, err := s.host.supportsProtocol(peer, info.protocolID)
+	if err != nil {
+		logger.Criticalf("could not check if protocol %s is supported by peer %s: %s", info.protocolID, peer, err)
+	} else if !support {
+		logger.Criticalf("protocol %s is not supported by peer %s", info.protocolID, peer)
```

Comment:
Should these be logged at the critical level? Why?

Reply:
fixed